flytekit/core/base_task.py

```diff
@@ -559,6 +559,15 @@ def task_config(self) -> Optional[T]:
         """
         return self._task_config
 
+    def get_error_file_name_suffix(self) -> Optional[str]:
```

Comment:
Nit (can be ignored right now until we are sure of how to implement this feature):
We shouldn't mention specific plugins like elastic here but rather speak of plugins using multiple workers which might produce independent error files.